resources/dicts/events/death/beach.json

```
@@ -0,0 +1,846 @@
+    {
+        "event_id": "bch_death_heat_multiapp2",
+        "location": ["beach"],
+        "season": [
+            "greenleaf",
+            "newleaf",
+            "leaf-fall"
+        ],
+        "tags": [],
+        "weight": 20,
+        "event_text": "r_c noticed m_c hadn't shown up for training again, yet after further prodding realized {PRONOUN/m_c/subject} hadn't been at camp for a few days. However, r_c never expected to find m_c like this: dead and dried out, their tattered pelt lifeless and dull.",
+        "m_c": {
+            "age": [
+                "adolescent"
+            ],
+            "status": [
+                "apprentice"
+            ],
+            "relationship_status": [
+                "app/mentor"
+            ],
+            "dies": true
+        },
+        "r_c": {
+            "age": [ "young adult",
+            "adult",
+            "senior adult" ],
+            "status": [ "any" ]
+        },
+        "history": [
+            {
+            "cats": [ "m_c" ],
+            "reg_death": "m_c died due to dehydration and overheating."
+            }
+        ]
+    },
+    {
+        "event_id": "bch_death_heat_multiapp3",
+        "location": ["beach"],
+        "season": [
+            "greenleaf",
+            "newleaf",
+            "leaf-fall"
+        ],
+        "tags": [],
+        "weight": 20,
+        "event_text": "r_c realizes {PRONOUN/r_c/subject} {VERB/r_c/haven't/hasn't} seen m_c around camp for a few days. Checking {PRONOUN/m_c/subject} favorite spots yields nothing, so r_c decides to go out and search for {PRONOUN/m_c/object} in the Clans' territory. However, r_c could never have expected to find m_c's shriveled-up body, clearly dead for some time now.",
+        "m_c": {
+            "age": [
+                "adolescent"
+            ],
+            "status": [
+                "apprentice"
+            ],
+            "relationship_status": [],
+            "dies": true
+        },
+        "r_c": {
+            "age": [
+                "young adult",
+                "adult",
+                "senior adult"
+            ],
+            "status": [ "any" ]
+        },
+        "history": [
+            {
+            "cats": [ "m_c" ],
+            "reg_death": "m_c died due to dehydration and heat exhaustion."
+            }
+        ]
+    },
+    {
+        "event_id": "bch_death_heat_multichild1",
+        "location": ["beach"],
+        "season": [
+            "greenleaf",
+            "newleaf",
+            "leaf-fall"
+        ],
+        "tags": [],
+        "weight": 20,
+        "event_text": "r_c is frantic with worry; m_c, {PRONOUN/r_c/poss} child, hadn't been seen around camp, no matter who {PRONOUN/r_c/subject} asked and where {PRONOUN/r_c/subject} searched. It's not until r_c stumbles across m_c's body, withered under the sun, that, with a despairing sound, {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} {PRONOUN/r_c/poss} greatest fears have come true.",
```

The second sentence here could probably benefit from rephrasing? There's just a lotta commas.

```suggestion
        "event_text": "r_c is frantic with worry; m_c, {PRONOUN/r_c/poss} child, hadn't been seen around camp, no matter who {PRONOUN/r_c/subject} asked and where {PRONOUN/r_c/subject} searched. It's not until r_c stumbles across m_c's body, withered under the sun, that {PRONOUN/r_c/subject} {VERB/r_c/realize/realizes} with a despairing sound {PRONOUN/r_c/poss} greatest fears have come true.",
```